### ISSUE TITLE
Web Export Tilemaps Do Not Render

### ISSUE THREAD

**Issue by cridenour** [godotengine/godot#66086]
### Godot version

4.0.beta.1

### System information

Web

### Issue description

HTML export will not display tilemaps. The same textures are shown when used in a Sprite2D.

### Steps to reproduce

Create a 2D Scene. Create a tilemap. Paint tilemap. Export to Web.

(Dance through CORS issues to get web export running locally)

See only the clear color.

### Minimal reproduction project

[tilemap-blank.zip](https://github.com/godotengine/godot/files/9595961/tilemap-blank.zip)

**Comment by cridenour** [comment]
Had the thought to run the editor in OpenGL and get this when opening the scene:

```
WARNING: blend shapes not supported by GLES3 renderer yet
     at: mesh_set_blend_shape_count (drivers/gles3/storage/mesh_storage.cpp:90)
ERROR: res://TileMapTest.tscn:6 - Parse Error: [ext_resource] referenced non-loaded resource at: res://icon.png
   at: _parse_ext_resource (scene/resources/resource_format_text.cpp:177)
ERROR: res://TileMapTest.tscn:6 - Parse Error: [ext_resource] referenced non-loaded resource at: res://icon.png
   at: load (scene/resources/resource_format_text.cpp:610)
ERROR: Failed loading resource: res://TileMapTest.tscn. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:221)
```

**Comment by Skwint** [comment]
I have this issue also, in 4.0 beta 2. I'd like to add that the TileMap itself must load in the web version because tile shaped collision detection still works, you just can't see it.

**Comment by zaevi** [comment]
Reproduced on latest master branch. May related to gles3 renderer.

**Comment by clayjohn** [comment]
I'm unable to reproduce this issue on the latest master 604abb434f6740935be669dc8152856412d9ca73

This is probably fixed by https://github.com/godotengine/godot/pull/68138. But it would be nice if someone who was able to reproduce the issue can confirm

**Comment by cridenour** [comment]
@clayjohn I can confirm this is closed with #68138 on my MRP and original project. Thank you!
